src/input-range-slider/input-range-slider.tsx

```diff
@@ -132,8 +136,24 @@ export const InputRangeSlider = ({
         );
     };
 
+    const formatIndicationLabel = () => {
+        return (
+            <LabelText>
+                Searching within&nbsp;
+                {indicationLabelPrefix}
+                {selection}
+                {indicationLabelSuffix}
+            </LabelText>
+        );
+    };
+
     return (
         <Wrapper {...otherProps}>
+            {showIndicationLabels && (
+                <IndicationLabelContainer>
+                    <div>{formatIndicationLabel()}</div>
```

Comment:
is this extra div needed?

Reply:
Removed, sry left this during the testing